clang/test/CodeGenSYCL/intel-restrict.cpp

```diff
@@ -10,19 +10,19 @@ int main() {
   int *b;
   int *c;
   kernel<class kernel_restrict>(
-      [a,b,c]() [[intel::kernel_args_restrict]] { c[0] = a[0] + b[0];});
```

Comment:
I modified this test in my previous commit and then reverted changes in this one. Looks like clang-format then make modifications as a result.

Reply:
I can leave this out. I think ClangFormat changed this because I touched this file in prior change and then reverted changes. I can try reverting this ClangFormat change but I am not sure if the bot will throw an error.

Reply:
Done